config/default.yml

```
@@ -49,6 +49,21 @@ Rails/ActionFilter:
   Include:
     - app/controllers/**/*.rb
 
+Rails/ActionOrder:
+  Description: 'Enforces consistent ordering of controller actions.'
```

```suggestion
  Description: 'Enforce consistent ordering of controller actions.'
```
(to match the language in other cop descriptions).